### PR TITLE
New development cycle

### PR DESCRIPTION
**Pull request by Tapchicoma**

### CODE REVIEW

File: plugin/build.gradle.kts [hunk 1]
```diff
@@ -68,13 +68,13 @@ fun setupPublishingEnvironment() {
         logger
             .info("`$keyProperty` or `$secretProperty` were not set. Attempting to configure from environment variables")
 
-        val key: String? = System.getProperty(keyEnvironmentVariable)
-        val secret: String? = System.getProperty(secretEnvironmentVariable)
+        val key: String? = System.getenv(keyEnvironmentVariable)
+        val secret: String? = System.getenv(secretEnvironmentVariable)
```
Comment (JLLeitschuh) [inline plugin/build.gradle.kts]: Thanks!

File: samples/kotlin-native/build.gradle.kts [hunk 2]
```diff
@@ -8,14 +8,14 @@ apply {
...
-kotlinNativeSourceSets["main"].component {
+kotlinNativeSourceSets["main"].component(Action {
     // https://github.com/JetBrains/kotlin-native/issues/1807
     if (this is KotlinNativeMainComponent) {
         outputKinds.set(listOf(OutputKind.EXECUTABLE))
         baseName.set("foo")
         extraOpts(listOf("-entry", "org.jlleitschuh.gradle.ktlint.sample.native.main"))
     }
-}
+})
```
Comment (JLLeitschuh) [inline samples/kotlin-native/build.gradle.kts]: Is this `Action` required now?? Gradle 4.10 shouldn't make this necessary.

Comment (Tapchicoma) [reply samples/kotlin-native/build.gradle.kts]: Gradle actually complained, as it can't detect second param type, so I has to use `Action {}`

Comment (JLLeitschuh) [reply samples/kotlin-native/build.gradle.kts]: Can you open an issue here:
https://github.com/gradle/kotlin-dsl/issues

Once done, leave a comment above this mentioning the issue link explaining `Action` is required.

I'll approve this PR after that.

Comment (Tapchicoma) [reply samples/kotlin-native/build.gradle.kts]: I don't think this is DSL problem, but rather native plugin itself:
```kotlin
e: /home/egorr/data/work/ktlint-gradle/samples/kotlin-native/build.gradle.kts:11:32: Overload resolution ambiguity: 
public open fun component(configureAction: Action<in AbstractKotlinNativeComponent>): KotlinNativeSourceSetImpl defined in org.jetbrains.kotlin.gradle.plugin.experimental.sourcesets.KotlinNativeSourceSetImpl
public open fun component(configureLambda: AbstractKotlinNativeComponent.() -> Unit): KotlinNativeSourceSetImpl defined in org.jetbrains.kotlin.gradle.plugin.experimental.sourcesets.KotlinNativeSourceSetImpl
```

Comment (JLLeitschuh) [reply samples/kotlin-native/build.gradle.kts]: Yea, it's a bug that the Gradle Kotlin DSL should be aware of and resolve correctly.

Comment (Tapchicoma) [reply samples/kotlin-native/build.gradle.kts]: See: https://github.com/gradle/kotlin-dsl/issues/1138

**Pull request merged**